config.py

```diff
@@ -56,6 +56,7 @@
 CHATROOM_PRESENCE = os.environ.get('ROOMS', '').split() or ROOMS_TO_JOIN
 
 ACCESS_CONTROLS = {'render test': {
-    'allowrooms': ('coala/cobot-test', 'coala/corobo',)}, }
+    'allowrooms': ('coala/cobot-test', 'coala/corobo',)},
+    'LabHub:create_issue_cmd': {'allowprivate': False}}
```

Comment:
FYI, in future, add a trailing comma and put the curly braces on the next line, so future additions dont need to modify your lines.